Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -127,22 +115,10 @@
 public class Metadaten {
     private static final Logger logger = LogManager.getLogger(Metadaten.class);
 
-    /**
-     * Get imageHelper.
-     *
-     * @return value of imageHelper
-     */
-    public ImagesHelper getImageHelper() {
```

Comment:
Maybe it is also possible to remove some code in ImagesHelper class, as main usage was in this class.

Reply:
There was no unused code in ImagesHelper class.